src/wasm-binary.h

```diff
@@ -155,7 +155,8 @@ enum Section {
   Functions = 2,
   DataSegments = 4,
   FunctionTable = 5,
-  End = 6
+  End = 6,
+  WLLSegment = 0x11
```

Comment:
Problem is that we have not agreed on a way to skip sections. They do not yet all have a size to allow them to be skipped. I was thinking that the WLL section might be an example, that all sections should have the same size at the start, but lets see. So for now it needs to be hard-coded in order to be able to skip it. It's not clear how long getting agreement on using the same size on all sections would take, so I can only appeal to accept this 'wart' in the interim. I would also like to be able to reserve a section, to have some table of registered sections - I don't want other projects dumping incompatible data in the same section (the decoder will need to be defensive and ignore invalid data, but still some cooperative system will help good agents).

Agreed to print a warning on unknown sections. I'll rework this PR to address the feedback, thank you.